app.js

```diff
@@ -2,87 +2,320 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
+"true" === true; //false
+//Thought 'true' is a truthy statement which is equal to true
+"new york"[0]; //n
+"new york"[1]; //e
+"san francisco"[2 * 3]; //a
+"engineering".indexOf("G"); //-1
+//Didn't know you can use [] to find indeces of a string
+//Didn't know you can use dot notation on a string
+
 // List 2 exprsssions you felt comfortable interpreting at a glance.
+10 % 2 === 0; //true
+21 % 2 === 0; //false
 
 // List 1 expression you are still 'shaky' on, why?
+"true" === true; //false
```

Comment:
`===` is looking to see if the type and value are equal. `'true'` is of type string and `true` is of type boolean.

If it said `'true' && true` then that would be true since both values are truthy